src/components/container/DailyDataChart/DailyDataChart.stories.tsx

```diff
@@ -51,7 +51,6 @@ export const stepsWithGapLineChart = {
         intervalType: "Week",
         weekStartsOn: "6DaysAgo",
```

Comment:
These value formatters were overriding the default ones, but we want to test the default ones in stories.